test/e2e/app_management_test.go

```diff
@@ -171,38 +171,42 @@ func TestManipulateApplicationResources(t *testing.T) {
 		Then().
 		Expect(SyncStatusIs(SyncStatusCodeSynced)).
 		And(func(app *Application) {
-			manifests, err := fixture.RunCli("app", "manifests", app.Name, "--source", "live")
-			assert.NoError(t, err)
-			resources, err := kube.SplitYAML(manifests)
-			assert.NoError(t, err)
+			manipulateApplicationResources(t, app)
```

Comment:
I actually don't think you should extract this func, I don't think you need it for your test. I'll explain why in my next comment.